src/commonMain/kotlin/com/github/quillraven/fleks/world.kt

```diff
@@ -26,24 +24,20 @@ fun wildcardSnapshotOf(components: List<Component<*>>, tags: List<UniqueId<*>>):
     return Snapshot(components as List<Component<out Any>>, tags as List<UniqueId<out Any>>)
 }
 
+typealias GenericWorld = World<*>
```

Comment:
In general I don't like such alias because it basically hides the generic syntax. When you see `World<Something>` it is obvious that it is a generic type while `GenericWorld` hides that and could be interpreted as something else.

Reply:
GenericWorld has been removed